kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/OperatorMain.java

```diff
@@ -33,12 +38,14 @@ public class OperatorMain {
     private static final Logger LOGGER = LoggerFactory.getLogger(OperatorMain.class);
     private MeterRegistry registry;
     private final Operator operator;
+    private HttpServer managementServer; // TODO HTTPS support
```

Comment:
```suggestion
    private final HttpServer managementServer; // TODO HTTPS support
```

Reply:
fixed as part of 969f6f9447b933eeb83c7e728bb289f7c21641ed